apiserver/facades/client/application/application.go

```diff
@@ -1666,3 +1666,19 @@ func (api *APIv6) unsetApplicationConfig(arg params.ApplicationUnset) error {
 	}
 	return nil
 }
+
+// Resolved implements the server side of Application.Resolved.
+func (api *APIv6) Resolved(p params.Resolved) error {
+	if err := api.checkCanWrite(); err != nil {
+		return err
+	}
+	if err := api.check.ChangeAllowed(); err != nil {
+		return errors.Trace(err)
+	}
+
+	unit, err := api.backend.Unit(p.UnitName)
```

Comment:
Add a TODO that we will either do the specified unit or "all".

Also, I see that this API is not a bulk API, ie it only accepts a params.Resolved containing one unit.
Since we are adding this as a new method on Application facade, we should take the opportunity to make it a bulk API. Introduce a params.UnitsResolved struct with a slice of entity tags and a bool (all). Both "omitempty". The caller will add either one or more unit tags or all=true (but not both). In practice, the CLI will continue to take a single unit arg and set up a single element entity slice. The result of the API will be a params.ErrorsResults. We use this pattern in many other places. On the api side, we have set patterns for processing the error results which can be copied. Be mindful of the slight changes needed though to accommodate the use of the all param.